src/common/ui/setting-check.vue

```diff
@@ -1,5 +1,10 @@
 <template>
-  <input type="checkbox" v-model="value" :disabled="disabled">
+  <label>
+    <input type="checkbox" v-model="value" :disabled="disabled">
+    <slot>
+      <span v-text="label" />
```

Comment:
I think the `<span>` here can be stripped, and the component can be used like this:

```html
<setting-check v-text="label" />
```

In this way we don't need a new prop `label` either.

Reply:
Yeah, looks tempting but it'll be a breaking change because it'll reduce the ability to customize these elements via custom CSS, like swapping elements via `order` trick, etc.

Reply:
Do you mean that CSS depends on the `<span>` element? I thought we can embed a `<span>` when we need it:

```html
<setting-check><span>label</span></setting-check>
```

It's OK to always keep a `<span>` though, this is not a problem.

Reply:
I didn't mean us, I meant the `custom CSS` in VM settings that some user might have used. On the other hand, this would be an extremely rare and possibly even weird thing to do so let's go with your idea.

Reply:
So I tried it and it didn't work: `v-text` replaces the entire contents, not just `slot`.

Reply:
`v-text` replaces the entire contents in `<setting-check>`, and the content will be put in `<slot>`.
Never mind, we should not break the custom CSS.